integration_tests/mockApis/remandAndSentencingApi.ts

```diff
@@ -1024,7 +1024,7 @@ export default {
               sentenceServeType: 'FORTHWITH',
               sentenceTypeId: '467e2fa8-fce1-41a4-8110-b378c727eed3',
               prisonId: 'MDI',
-              sentenceReference: '0',
+              sentenceReference: '71bb9f7e-971c-4c34-9a33-43478baee74f',
```

Comment:
just wondering why this had to change.. isn't sententenceReference (or sentenceUUID) fdifferent to charge uuid?!?
I may be getting confused by terminology!!